app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -24,6 +32,7 @@ import javax.inject.Inject
 @HiltViewModel
 class ConversationListViewModel @Inject constructor(
     private val navigationManager: NavigationManager,
+    private val getConversationUseCase: GetConversationsUseCase,
```

Comment:
Maybe we can call this var just `getConversations` so we can then invoke the function without the dot invoke -> `getConversations()` That way it looks just like any other function